integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkTableInfo.java

```diff
@@ -165,6 +167,32 @@ public List<SparkColumnInfo> getPartitionedColumns() {
         .collect(Collectors.toList());
   }
 
+  private static boolean isBucketPartition(boolean supportsBucketPartition, Transform transform) {
+    return supportsBucketPartition && !(transform instanceof SortedBucketTransform);
+  }
+
+  private static SparkTransformConverter getSparkTransformConverter(Table baseTable) {
```

Comment:
it's a little hack here

Reply:
How about this? cc @FANNG1 

```
public interface Basetable {
  
  SparkTransformConverter getSparkTransformConverter();
  
}
```


```
public class SparkIcebergTable extends SparkTable implements Basetable{
  
  ...
  
  @Override
  public SparkTransformConverter getSparkTransformConverter() {
    return gravitinoTableInfoHelper.getSparkTransformConverter();
  }
  
  ...
}
```


```
public class SparkHiveTable extends HiveTable implements Basetable{
  
  ...

  @Override
  public SparkTransformConverter getSparkTransformConverter() {
    return gravitinoTableInfoHelper.getSparkTransformConverter();
  }
  
  ...
}
```

Reply:
let me think a while

Reply:
I couldn't figure out a general way, How about do a hard code about Hive supports bucket transform partition and Iceberg not support since it's just test code.  The current implement and interface way is a heavy for me.

Reply:
paimon only support Identity partition, so it is ok for me.